src/Mod/Path/InitGui.py

```diff
@@ -129,7 +129,8 @@ def Initialize(self):
                 threedcmdgroup = ['Path_3dTools']
                 FreeCADGui.addCommand('Path_3dTools', PathCommandGroup(threedopcmdlist, QtCore.QT_TRANSLATE_NOOP("Path", '3D Operations')))
             except ImportError:
-                FreeCAD.Console.PrintError("OpenCamLib is not working!\n")
+                if not PathPreferences.suppressOpenCamLibWarning():
```

Comment:
Was there supposed to be a third preference setting to suppress OCL warnings?  Do we need individually disableable warnings.  Maybe just one for 'enable verbose warnings'  and default it to off so the default condition is to hide this stuff.

Reply:
OCL is not in the preferences because it's only part if "experimental features" is enabled. Didn't think that was worth the effort.